src/main/java/mate/jdbc/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.jdbc.exceptions;
```

Comment:
```suggestion
package mate.jdbc.exception;
```
better use singular for package names